tests/phpunit/tests/fonts/font-library/wpFontFamily/install.php

```diff
@@ -0,0 +1,562 @@
+						'tmp_name' => wp_tempnam( 'Lato-' ),
+						'error'    => 0,
+						'size'     => 123,
+					),
+					'files2' => array(
+						'name'     => 'lato2.woff',
+						'type'     => 'font/woff',
+						'tmp_name' => wp_tempnam( 'Lato-' ),
+						'error'    => 0,
+						'size'     => 123,
+					),
+				),
+				'expected'   => array( 'lato_normal_400.ttf', 'lato_normal_500.ttf' ),
+			),
+		);
+	}
+
+	public function test_should_overwrite_fontface_with_different_extension() {
+		$font_data_initial    = array(
+			'name'       => 'Inter',
+			'slug'       => 'inter',
+			'fontFamily' => 'Inter',
+			'fontFace'   => array(
+				array(
+					'fontFamily'   => 'Inter',
+					'fontStyle'    => 'italic',
+					'fontWeight'   => '500',
+					'uploadedFile' => 'files0',
+				),
+				array(
+					'fontFamily'   => 'Inter',
+					'fontStyle'    => 'italic',
+					'fontWeight'   => '900',
+					'uploadedFile' => 'files1',
+				),
+			),
+		);
+		$files_data_initial   = array(
+			'files0' => array(
+				'name'     => 'inter1.ttf',
+				'type'     => 'font/woff',
+				'tmp_name' => wp_tempnam( 'Inter-' ),
+				'error'    => 0,
+				'size'     => 123,
+			),
+			'files1' => array(
+				'name'     => 'inter1.woff',
+				'type'     => 'font/woff',
+				'tmp_name' => wp_tempnam( 'Inter-' ),
+				'error'    => 0,
+				'size'     => 123,
+			),
+		);
+		$font_data_overwrite  = array(
+			'name'       => 'Inter',
+			'slug'       => 'inter',
+			'fontFamily' => 'Inter',
+			'fontFace'   => array(
+				array(
+					'fontFamily'   => 'Inter',
+					'fontStyle'    => 'italic',
+					'fontWeight'   => '500',
+					'uploadedFile' => 'files0',
+				),
+				array(
+					'fontFamily'   => 'Inter',
+					'fontStyle'    => 'italic',
+					'fontWeight'   => '900',
+					'uploadedFile' => 'files1',
+				),
+			),
+		);
+		$files_data_overwrite = array(
+			'files0' => array(
+				'name'     => 'inter1.woff',
+				'type'     => 'font/woff',
+				'tmp_name' => wp_tempnam( 'Inter-' ),
+				'error'    => 0,
+				'size'     => 123,
+			),
+			'files1' => array(
+				'name'     => 'inter1.ttf',
+				'type'     => 'font/ttf',
+				'tmp_name' => wp_tempnam( 'Inter-' ),
+				'error'    => 0,
+				'size'     => 123,
+			),
+		);
+
+		$expected = array( 'inter_italic_500.woff', 'inter_italic_900.ttf' );
+
+		// Set up the temporary files.
+		foreach ( $files_data_initial as $file ) {
+			if ( 'font/ttf' === $file['type'] ) {
+				copy( __DIR__ . '/../../../data/fonts/Merriweather.ttf', $file['tmp_name'] );
+			} elseif ( 'font/woff' === $file['type'] ) {
+				copy( __DIR__ . '/../../../data/fonts/cooper-hewitt.woff', $file['tmp_name'] );
+			}
+		}
+		foreach ( $files_data_overwrite as $file ) {
+			if ( 'font/ttf' === $file['type'] ) {
+				copy( __DIR__ . '/../../../data/fonts/Merriweather.ttf', $file['tmp_name'] );
+			} elseif ( 'font/woff' === $file['type'] ) {
+				copy( __DIR__ . '/../../../data/fonts/cooper-hewitt.woff', $file['tmp_name'] );
+			}
+		}
```

Comment:
Test directory const.

Reply:
Thanks for flagging that. Done here: https://github.com/WordPress/wordpress-develop/pull/5285/commits/e218ec8b94e679da12c013bf7ce617f169ee0738

Reply:
Confirmed this was indeed resolved by https://github.com/WordPress/wordpress-develop/commit/e218ec8b94e679da12c013bf7ce617f169ee0738. However, the `path_join()` is unnecessary. Will commit on and fix the new code to be consistent within Core's tests.

Reply:
Confirmed that https://github.com/WordPress/wordpress-develop/pull/5285/commits/e218ec8b94e679da12c013bf7ce617f169ee0738 did indeed resolve this. I did a follow-up commit https://github.com/WordPress/wordpress-develop/pull/5285/commits/d6534cc08a4262f02c858d59a7e1a1d4c945b0b5 to remove the `path_join()` to simplify and be consistent in Core tests ✅ 

Making this as resolved, which will fold / hide it. I don't think leaving it unresolved helps to provide additional context.